BE/sidedish11/src/main/java/dev/codesquad/java/sidedish11/entity/Category.java

```diff
@@ -0,0 +1,63 @@
+package dev.codesquad.java.sidedish11.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter @Setter
+@ToString
+public class Category {
+    @Id
+    private Long id;
+    private String title;
+    private String name;
+    private String description;
+    private List<Item> items = new ArrayList<>();
+
+    public Category() {
+    }
```

Comment:
빈 중괄호는 한 줄에 선언해 주세요. 

```suggestion
public Category() {}